diff -u a/cordova-plugin-file/CDVFile.h b/cordova-plugin-file/CDVFile.h
--- a/cordova-plugin-file/CDVFile.h	2015-05-08 00:42:13.000000000 +0800
+++ b/cordova-plugin-file/CDVFile.h	2015-05-08 00:55:23.000000000 +0800
@@ -20,8 +20,8 @@
 #import <Foundation/Foundation.h>
 #import <Cordova/CDVPlugin.h>
 
-NSString* const kCDVAssetsLibraryPrefix;
-NSString* const kCDVFilesystemURLPrefix;
+extern NSString* const kCDVAssetsLibraryPrefix;
+extern NSString* const kCDVFilesystemURLPrefix;
 
 enum CDVFileError {
     NO_ERROR = 0,
