diff -u a/cordova-plugin-dialogs/CDVNotification.m b/cordova-plugin-dialogs/CDVNotification.m
--- a/cordova-plugin-dialogs/CDVNotification.m	2015-05-12 14:51:45.000000000 +0800
+++ b/cordova-plugin-dialogs/CDVNotification.m	2015-05-12 14:53:14.000000000 +0800
@@ -186,7 +186,7 @@
 static void playBeep(int count) {
     SystemSoundID completeSound;
     NSInteger cbDataCount = count;
-    NSURL* audioPath = [[NSBundle mainBundle] URLForResource:@"CDVNotification.bundle/beep" withExtension:@"wav"];
+    NSURL* audioPath = [[NSBundle bundleForClass:[CDVNotification class]] URLForResource:@"CDVNotification.bundle/beep" withExtension:@"wav"];
     #if __has_feature(objc_arc)
         AudioServicesCreateSystemSoundID((__bridge CFURLRef)audioPath, &completeSound);
     #else
